Assets/Scripts/Models/Tile.cs

```diff
@@ -112,7 +112,7 @@ public float PathfindingCost
 
     public int Y { get; private set; }
 
-    public float MovmentModifier { get; set; }
+    public float MovementModifier { get; set; }
```

Comment:
This should either be a proper accessor, with a private or protected set, or just a variable, without the {get;set}